pulsar-client-cpp/python/CMakeLists.txt

```diff
@@ -48,8 +48,17 @@ endif()
 set(PYTHON_WRAPPER_LIBS ${Boost_PYTHON_LIBRARY} ${Boost_PYTHON3_LIBRARY}
                         ${Boost_PYTHON27-MT_LIBRARY} ${Boost_PYTHON37-MT_LIBRARY})
 
+if (APPLE)
+    set(PYTHON_WRAPPER_LIBS ${PYTHON_WRAPPER_LIBS}
+                         ${Boost_PYTHON27-MT_LIBRARY_RELEASE} ${Boost_PYTHON37-MT_LIBRARY_RELEASE})
+endif()
```

Comment:
Recently I found some build issues like #11747. It's caused by `Boost_<component>-MT_LIBRARY_RELEASE` is set with `Boost_<component>-MT_LIBRARY_RELEASE-NOTFOUND` and added to `PYTHON_WRAPPER_LIBS`.

Could you explain why this line is needed for macOS? @merlimat

Reply:
BTW, here is a fix: https://github.com/apache/pulsar/pull/13193. But IMO these code could be removed.